roles/pulibrary.figgy/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']
+).get_hosts('all')
+
+
+def test_hosts_file(host):
```

Comment:
We should probably add a couple tests, but that would not need to be done here...

Reply:
added some tests.